src/main/java/com/github/manolo8/darkbot/gui/tree/EditorProvider.java

```diff
@@ -4,21 +4,7 @@
 import com.github.manolo8.darkbot.config.ImageWrapper;
 import com.github.manolo8.darkbot.config.tree.ConfigField;
 import com.github.manolo8.darkbot.config.types.Editor;
-import com.github.manolo8.darkbot.gui.tree.editors.BooleanEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.CharacterEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.ColorEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.ConditionEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.DropdownEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.FontEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.ImagePicker;
-import com.github.manolo8.darkbot.gui.tree.editors.MultiDropdownEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.NumberEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.PercentEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.PlayerTagEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.RangeEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.ShipModeEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.StringEditor;
-import com.github.manolo8.darkbot.gui.tree.editors.TableEditor;
+import com.github.manolo8.darkbot.gui.tree.editors.*;
```

Comment:
Disable wildcard imports in your IDE, keep them separated, it is generally bad to use wildcard imports in java because they can cause code to break for none of your fault (a class being added elsewhere, that now is imported and matches-up with a pre-existing name)